multi-container/hello-nginx-sample/tests/e2e-test.sh

```diff
@@ -40,7 +40,7 @@ then
 fi
 
 # Check Cloud Run MC logs for signs of successful request to hello container.
-MC_HELLO_LOG=$(gcloud logging read "resource.type=cloud_run_revision AND resource.labels.service_name=${MC_SERVICE_NAME} AND labels.container_name=hello" --limit 1 | grep -e 'Hello from Cloud Run')
+MC_HELLO_LOG=$(gcloud logging read "resource.type=cloud_run_revision AND resource.labels.service_name=${MC_SERVICE_NAME} AND labels.container_name=hello" --limit 2 | grep -e 'Hello from Cloud Run')
```

Comment:
Note: In `tests/e2e-test.sh`, updated result `limit` to 2 to accomodate new startup tcp probe logs now displaying before expected service.

(`textPayload: STARTUP TCP probe succeeded after 1 attempt for container "hello" on
  port 8888.`)